content/docs/command-reference/run.md

```diff
@@ -106,6 +106,10 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
+- Missing outputs can be
+  [added to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  without running it again.
```

Comment:
```suggestion
  without executing it again.
```
^ Note that we prefer that term since you can "run" stages either with `dvc run` or with `dvc repro` (sometimes even with `dvc pull`).